app/src/main/res/values/strings.xml

```diff
@@ -1870,4 +1870,81 @@
+    <string name="recommended_reading_list_interest_source_message">How should we build your recommended reading list?</string>
+    <string name="recommended_reading_list_interest_source_interests">Let me choose based on my interests</string>
+    <string name="recommended_reading_list_interest_source_saved">Choose for me based on my saved articles</string>
+    <string name="recommended_reading_list_interest_source_history">Choose for me based on my reading history</string>
+    <string name="recommended_reading_list_interest_pick_title">What are you interested in?</string>
+    <string name="recommended_reading_list_interest_pick_search_hint">Search for an article</string>
+    <string name="recommended_reading_list_interest_pick_selected_articles">%d selected</string>
+    <string name="recommended_reading_list_interest_pick_random_button_content_description">Randomly select</string>
+    <string name="recommended_reading_list_interest_pick_random_snackbar">We\'ve randomly selected a few articles for you.</string>
+    <string name="recommended_reading_list_interest_pick_random_snackbar_action">Undo</string>
+    <string name="recommended_reading_list_interest_pick_random_dialog_title">Are you sure you want to randomize your selection?</string>
+    <string name="recommended_reading_list_interest_pick_random_dialog_message">This will clear your current selections and choose new articles at random.</string>
+    <string name="recommended_reading_list_interest_pick_random_dialog_positive_button">Yes, choose for me</string>
+    <string name="recommended_reading_list_interest_pick_random_dialog_negative_button">Never mind</string>
+    <string name="recommended_reading_list_page_subtitle_made_for">Made for %s</string>
+    <plurals name="recommended_reading_list_page_subtitle_articles">
+        <item quantity="one">• %d article</item>
+        <item quantity="other">• %d articles</item>
+    </plurals>
+    <string name="recommended_reading_list_page_description_daily">Your daily reading list. Learn about new topics,  picked just for you. Updates at midnight.</string>
+    <string name="recommended_reading_list_page_description_weekly">Your weekly reading list. Learn about new topics, picked just for you. Updates every Monday.</string>
+    <string name="recommended_reading_list_page_description_monthly">Your monthly reading list. Learn about new topics,  picked just for you. Updates on the first of each month.</string>
+    <string name="recommended_reading_list_page_overflow_save_list">Save list</string>
+    <string name="recommended_reading_list_page_overflow_customize">Customize</string>
+    <string name="recommended_reading_list_page_overflow_about">About this feature</string>
+    <string name="recommended_reading_list_page_snackbar">Discover will update with five new articles each %s. Your recommendations are stored locally on your device.</string>
```

Comment:
It looks like we're hardcoding `five` here.. Isn't it configurable?

Also, what is `each %s` replaced by? If this is supposed to be `day` / `week` / `month`, should we have those strings here for translation?

Reply:
Done